streams/src/main/java/org/apache/kafka/streams/processor/internals/metrics/TaskMetrics.java

```diff
@@ -128,6 +131,22 @@ public static Sensor activeBufferedRecordsSensor(final String threadId,
         return sensor;
     }
 
+    public static Sensor totalBytesSensor(final String threadId,
```

Comment:
While reviewing the PR, I feel the name `total-bytes` under

```
type = stream-task-metrics
thread-id = [thread ID]
task-id = [task ID]
```

is a bit too vague, what about renaming it to `total-input-buffer-bytes`. WDYT @vamossagar12 @ableegoldman ? If we agree here we'd need to update the KIP as well.

Reply:
That makes sense to me. WDYT @ableegoldman ?

Reply:
Btw I am more or less done with all other changes. So if we can decide on this, I would update it. There was another thing which I had missed with update partitions in partition group where if a partition is removed, we needed to remove the corresponding buffer size of it.

Reply:
cc @ableegoldman let me know WDYT about the new metric name?

Reply:
hey @ableegoldman , the list of conflicting files is growing on this PR :D Whenever you get the chance could you plz share your thoughts on this and a couple of other PRs ([this](https://github.com/apache/kafka/pull/11211) and [this](https://github.com/apache/kafka/pull/11433)

Reply:
hi.. bumping this thread again

Reply:
@ableegoldman ping :)

@vamossagar12 I'm just back from a long vacation and is reloading all my review memory here :) is this ready for another review now besides this comment for naming? If yes could you rebase and ping me?

Reply:
@guozhangwang Welcome back :) From my side, yeah I think it's ready for another review besides the comment. I have rebased the changes as well.